conf/standalone.conf

```diff
@@ -268,3 +268,14 @@ keepAliveIntervalSeconds=30
 
 # How often broker checks for inactive topics to be deleted (topics with no subscriptions and no one connected)
 brokerServicePurgeInactiveFrequencyInSeconds=60
+
+### --- WebSocket --- ###
+
+# Enable the WebSocket API service in broker
+webSocketServiceEnabled=false
+
+# Number of IO threads in Pulsar Client used in WebSocket proxy
+# webSocketNumIoThreads=8
```

Comment:
I would leave the lines uncommented here (even though we don't know the number of CPU cores at this point), for consistency with all the other variables. 

Also, when running within docker container, the configuration script that allows you to override the config is based on apply env variable to existing keys in the conf files. If the keys are commented, that won't work.